ext/financialacls/financialacls.php

```diff
@@ -384,6 +384,23 @@ function financialacls_civicrm_alterMenu(array &$menu): void {
   $menu['civicrm/admin/financial/financialType']['access_arguments'] = [['administer CiviCRM Financial Types']];
 }
 
+/**
+ * @param string $formName
+ * @param \CRM_Core_Form $form
+ */
+function financialacls_civicrm_preProcess(string $formName, \CRM_Core_Form $form): void {
+  if (!financialacls_is_acl_limiting_enabled()) {
+    return;
+  }
+  if (str_starts_with($formName, 'CRM_Contribute_Form_Contribution')) {
```

Comment:
hmm so this would also cover the Backoffice form as well as frontend forms?

Reply:
maybe add an _ here at the end to make sure we just capture front end forms?

Reply:
@seamuslee001 I've added the `_` per your original suggestion

I think there is a pre-existing but in the backoffice form & it will require unravelling those actions to fix so I scaled this back to the form I'm working on.